### ISSUE TITLE
During my first update using the utility, Debconf on pop-os appeared and froze.

### ISSUE THREAD

**Issue by RockChalkBogota** [pop-os/iso#112]
If this is a bug, please use the template below. If this is a feature request, question, or a general discussion topic - please post on our subreddit https://reddit.com/r/pop_os - as that is the proper forum for those types of posts.

**Reporter info**

```
 * Distribution - (run ```lsb_release -a```)
 * Related Application and/or Package Version - apt policy $PACKAGE NAME
```
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu Artful Aardvark (development branch)
Release:	17.10
Codename:	artful

**Issue/Bug Description**
Hello, I just installed this OS. I ran an update using the Pop Shop utility. During the update, Debconf on pop-os appeared asking for the country of origin for my keyboard.  I thought this was strange since I set it up at the beginning of the install.  Anyway, I clicked forward, since it was the only option I had, and it froze that window. I had to force quit the window. 

**Steps to reproduce (if you know)**
Using the Pop Shop to update? Currently using a Das Keyboard if that makes a difference. 

**Expected behaviour**
Not to prompt for keyboard zoning.

**Other Notes**
Cool stuff! Great seeing this come out of Denver.  Shout out from Kansas City!

**Comment by Shamash2014** [comment]
Can confirm, even on second update it freezes

**Comment by WatchMkr** [comment]
Which language and KB layouts are you using during installation?

**Comment by RockChalkBogota** [comment]
The default ones. I think it was English and English.

**Comment by Shamash2014** [comment]
en_US

**Comment by sysdrum** [comment]
I can confirm the bug as well. 
en_US | QWERTY keyboard

**Comment by sarahpandabeara** [comment]
@jackpot51 this happened with me as well on the NVIDIA Beta. Happy to work through what I did (or didn't) do live if need be

**Comment by Shamash2014** [comment]
@jackpot51 Real hardware, 8GB, 300GB for root and 700GB for home, build 27

**Comment by sarahpandabeara** [comment]
@brs17 please try to reproduce

**Comment by isantop** [comment]
@brs17 Steps to reproduce: Any time debconf opens during an update in Pop_Shop, Pop_shop will freeze.

**Comment by brs17** [comment]
We (@djordan2 and I ) were able to recreate the issue of Debconf crashing when prompting the user during an update. 

We believe it is not a bug that debconf asks questions regarding what kind of keyboard layout you would like. If the updated package and what is set on your machine is different, then debconf will launch requesting the user to specify which setting is desired. We were able to recreate the issue with grub, changing a setting on the machine and updating grub. 

Recreating the issue with grub:
![screenshot from 2017-10-12 15-53-34](https://user-images.githubusercontent.com/7460774/31521734-dfc6cd40-af67-11e7-962a-f45636a02814.png)

Selecting the "Forward" button freezes Debconf
![screenshot from 2017-10-12 15-54-42](https://user-images.githubusercontent.com/7460774/31521760-f79d5088-af67-11e7-8085-83e5f449e80e.png)

Debconf crashes
![screenshot from 2017-10-12 15-54-53](https://user-images.githubusercontent.com/7460774/31521769-068bbae4-af68-11e7-899f-e2f9fe816b4b.png)

We did see the the updates that were running complete in the background (Pop!_Shop)
